### PR TITLE
folder structure

### PR DESCRIPTION
**Pull request by MadhushaPrasad**
There is no folder structure of the main branch. So If this is right please merge this.

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"D:\\MS_Club\\msclubwebsite\\src\\index.js":"1","D:\\MS_Club\\msclubwebsite\\src\\reportWebVitals.js":"2","D:\\MS_Club\\msclubwebsite\\src\\App.js":"3"},{"size":517,"mtime":1617766371217,"results":"4","hashOfConfig":"5"},{"size":375,"mtime":1617766371218,"results":"6","hashOfConfig":"5"},{"size":553,"mtime":1617766371216,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1534fby",{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"D:\\MS_Club\\msclubwebsite\\src\\index.js",[],"D:\\MS_Club\\msclubwebsite\\src\\reportWebVitals.js",[],"D:\\MS_Club\\msclubwebsite\\src\\App.js",[]]
```
Comment (Shehanka) [inline .eslintcache:1]: gitignore this

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="JSX" />
+  </component>
+</project>
```
Comment (Shehanka) [inline .idea/misc.xml:6]: gitignore and remove all .idea from the git cache

**Review by Shehanka — CHANGES_REQUESTED** [review]
All component js files and CSS files should start with capital

**Comment by lwpamihiranga** [comment]
@MadhushaPrasad let's change the branch name as feature/<specific issue name>. otherwise, it will not be consistent 🙂

**Review by lwpamihiranga — CHANGES_REQUESTED** [review]
@MadhushaPrasad  let's change the branch name to `feature/<specific issue name>`. otherwise, it will not be consistent 🙂

**Pull request closed without merging**